akka-docs/src/main/paradox/java/stream/stream-introduction.md

```diff
@@ -1,75 +1 @@
-formulate stream processing setups such that we can then execute them
-efficiently and with bounded resource usage—no more OutOfMemoryErrors. In order
-to achieve this our streams need to be able to limit the buffering that they
-employ, they need to be able to slow down producers if the consumers cannot
-keep up. This feature is called back-pressure and is at the core of the
-[Reactive Streams](http://reactive-streams.org/) initiative of which Akka is a
-founding member. For you this means that the hard problem of propagating and
-reacting to back-pressure has been incorporated in the design of Akka Streams
-already, so you have one less thing to worry about; it also means that Akka
-Streams interoperate seamlessly with all other Reactive Streams implementations
-(where Reactive Streams interfaces define the interoperability SPI while
-implementations like Akka Streams offer a nice user API).
-
-### Relationship with Reactive Streams
-
-The Akka Streams API is completely decoupled from the Reactive Streams
-interfaces. While Akka Streams focus on the formulation of transformations on
-data streams the scope of Reactive Streams is just to define a common mechanism
-of how to move data across an asynchronous boundary without losses, buffering
-or resource exhaustion.
-
-The relationship between these two is that the Akka Streams API is geared
-towards end-users while the Akka Streams implementation uses the Reactive
-Streams interfaces internally to pass data between the different processing
-stages. For this reason you will not find any resemblance between the Reactive
-Streams interfaces and the Akka Streams API. This is in line with the
-expectations of the Reactive Streams project, whose primary purpose is to
-define interfaces such that different streaming implementation can
-interoperate; it is not the purpose of Reactive Streams to describe an end-user
-API.
-
-## How to read these docs
-
-Stream processing is a different paradigm to the Actor Model or to Future
-composition, therefore it may take some careful study of this subject until you
-feel familiar with the tools and techniques. The documentation is here to help
-and for best results we recommend the following approach:
-
- * Read the @ref:[Quick Start Guide](stream-quickstart.md#stream-quickstart) to get a feel for how streams
-look like and what they can do.
- * The top-down learners may want to peruse the @ref:[Design Principles behind Akka Streams](../general/stream/stream-design.md) at this
-point.
- * The bottom-up learners may feel more at home rummaging through the
-@ref:[Streams Cookbook](stream-cookbook.md).
- * For a complete overview of the built-in processing stages you can look at the
-table in @ref:[stages overview](stages-overview.md)
- * The other sections can be read sequentially or as needed during the previous
-steps, each digging deeper into specific topics.
\ No newline at end of file
+../../scala/stream/stream-introduction.md
```

Comment:
I'm sorry, this is a bit confusing in the GitHub UI, but this is still not a symlink, just a file with `../../scala/stream/stream-introduction.md` in it... you can test the documentation output with 'sbt paradox', then you'll see this page doesn't have the expected contents but just this filename.

I'll add a short guide on creating symlinks to #23052 tomorrow!